jax/numpy/lax_numpy.py

```diff
@@ -2625,6 +2625,25 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
 
+@_wraps(onp.corrcoef)
+def corrcoef(x, y=None, rowvar=True, bias=None, ddof=None):
+  c = cov(x, y, rowvar)
+  if isscalar(c) or (len(shape(c)) != 1 and len(shape(c)) != 2):
```

Comment:
JAX doesn't distinguish between scalars and 0D arrays. Perhaps there is value in testing for both (e.g., if a numpy classic array is passed here).

Why the two length tests? Why not `len(shape(c) == 0`?

Reply:
Fixed.